pkg/acquisition/modules/appsec/appsec_runner.go

```diff
@@ -45,6 +46,8 @@ func (r *AppsecRunner) MergeDedupRules(collections []appsec.AppsecCollection, lo
 				logger.Debugf("Discarding duplicate rule : %s", rule)
 			}
 		}
+		// Don't mess up with native modsec rules
+		rulesArr = append(rulesArr, collection.NativeRules...)
```

Comment:
If I understand correctly, you should make the append after the logger.Warningf() to avoid telling misleading information to users (ie count the NativeRules in excess for the duplicates)